apis/infix-en_GB/extensions/jdk8/atrium-api-infix-en_GB-jdk8-jvm/src/test/kotlin/ch/tutteli/atrium/api/infix/en_GB/jdk8/PathFeatureAssertionsSpec.kt

```diff
@@ -1,15 +1,15 @@
 package ch.tutteli.atrium.api.infix.en_GB.jdk8
 
 import ch.tutteli.atrium.creating.Expect
-import ch.tutteli.atrium.specs.fun1
-import ch.tutteli.atrium.specs.notImplemented
-import ch.tutteli.atrium.specs.property
+import ch.tutteli.atrium.specs.*
 import ch.tutteli.atrium.specs.testutils.WithAsciiReporter
 import java.nio.file.Path
 
 class PathFeatureAssertionsSpec : ch.tutteli.atrium.specs.integration.PathFeatureAssertionsSpec(
     property<Path, Path>(Expect<Path>::parent),
     fun1<Path, Expect<Path>.() -> Unit>(Expect<Path>::parent),
+    feature1<Path, String, Path>(Expect<Path>::resolve),
+    notImplemented(),
```

Comment:
I cannot implement the infix function with both parameters (String and lambda), so I am using notImplemented here. Not sure if it's the right thing to do.

Reply:
It's ok that you cannot implement the infix for two parameters (which is not possible). I am not yet sure how I am going to support this in the new infix API. However, you cannot use notImplemented() as this will fail the test.
Instead, create the following function in the same file
```
private fun <T: Path> resolve(expect: Expect<T>, other: String): Expect<T> = and {
  expect resolve other
}
```  
and use it
```suggestion
    "resolve with assertionCreator not implemented in this API" to ::resolve,
```